finances/models.py

```diff
@@ -599,7 +599,7 @@ def pay(self, operator, recipient):
 
         # Calcul du prix par weight
         total_weight = self.get_total_weights_participants()
-        final_price_per_weight = round(self.price / total_weight, 2)
+        final_price_per_weight = round(total_price / total_weight, 2)
```

Comment:
Check ZeroDivision here ?

Reply:
Well, currently total_weight can't be null (it is checked in dispatch view).
I added the check "just to be sure"

Reply:
Fix: 91273568d48f65078cbc4a51ea84f20e78b18b1f